modules/cache/variables.tf

```
@@ -40,7 +40,7 @@ variable "tags" {
 }
 
 variable "create_cache_bucket" {
-  description = "This module is by default included in the runner module. To disable the creation of the bucket this parameter can be disabled."
+  description = "(deprecated) If the cache should not be craeted, remove the whole module call!"
   type        = bool
   default     = true
```

With a validation rule like below the user gets some nice feedback if still using the variable.
```suggestion
  default     = null
  
  validation {
    condition     = anytrue([var.create_cache_bucket == null])
    error_message = "Deprecated, don't call the module when not creating a cache bucket."
  }
```

---

Very good!